src/autofill-utils.js

```diff
@@ -294,10 +294,13 @@ const isLikelyASubmitButton = (el) => {
     const value = (el instanceof HTMLInputElement ? el.value || '' : '')
     const contentExcludingLabel = text + ' ' + title + ' ' + value
 
-    return (el.getAttribute('type') === 'submit' || // is explicitly set as "submit"
-    /primary|submit/i.test(el.className) || // has high-signal submit classes
-    SUBMIT_BUTTON_REGEX.test(contentExcludingLabel) || // has high-signal text
-    (el.offsetHeight * el.offsetWidth >= 10000 && !/secondary/i.test(el.className))) && // it's a large element 250x40px
+    return (
+        el.getAttribute('type') === 'submit' || // is explicitly set as "submit"
+        /primary|submit/i.test(el.className) || // has high-signal submit classes
+        SUBMIT_BUTTON_REGEX.test(contentExcludingLabel) || // has high-signal text
+        (el.offsetHeight * el.offsetWidth >= 10000 && !/secondary/i.test(el.className)) // it's a large element 250x40px
+    ) &&
+    el.offsetHeight * el.offsetWidth >= 2000 && // it's not a very small button like inline links and such
```

Comment:
Just adding this condition (the comment explains it) and applying saner formatting for the previous part.

Reply:
❓ Does one of the added test cases in this PR cover this change?

Reply:
Thanks for raising. In fact, it wasn't, but since this is a rather obscure case and it was pretty time-consuming to reproduce on Eventbrite, I decided to add the infrastructure necessary to test this. It's a minor improvement, but significant and I've added the docs for this stuff. Here's the [whole commit](https://github.com/duckduckgo/duckduckgo-autofill/pull/231/commits/40f487f45695f61ceb81419c662aad1ab49650d8).

Note that I had to add a specific fake form to test this, because the test was still passing on the Eventbrite form because of all the other improvements in this PR 💪.